engine/plugins/server/server.js

```diff
@@ -92,21 +93,38 @@ module.exports = function(options, imports, register) {
         });
 
         /**
-         * FBS have been detected to be offline.
+         * Handle FBS is offline events.
          */
         bus.on('fbs.offline', () => {
             //
             // @TODO: Make the frontend react to this event.
             //
+            // socket.emit('Offline');
         });
 
         /**
-         * FBS have been detected as online.
+         * Handle FBS is online events.
          */
         bus.on('fbs.online', () => {
             //
             // @TODO: Make the frontend react to this event.
             //
+            // socket.emit('Online');
```

Comment:
Why commment this out? Relation to refresh token?

Reply:
Will remove... Is not related to feature.